.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -0,0 +1,460 @@
+<%
+const UNIT_TESTS_BUILD_VARIANTS = [
+  {
+    name: 'unit-tests-ubuntu',
+    display_name: 'Unit Tests Ubuntu',
+    run_on: 'ubuntu2004-large',
+  },
+  {
+    name: 'unit-tests-windows',
+    display_name: 'Unit Tests Windows',
+    run_on: 'windows-vsCurrent-large',
+  },
+  {
+    name: 'unit-tests-rhel',
+    display_name: 'Unit Tests RHEL',
+    run_on: 'rhel80-large',
+  },
+  {
+    name: 'unit-tests-macos',
+    display_name: 'Unit Tests MacOS',
+    run_on: 'macos-14-arm64',
+    run_on_gui: 'macos-14-arm64-gui'
+  }
+];
+
+const PACKAGE_BUILD_VARIANTS = [
+  {
+    name: 'package-ubuntu',
+    display_name: 'Package Ubuntu',
+    run_on: 'ubuntu1604-large',
+  },
+  {
+    name: 'package-windows',
+    display_name: 'Package Windows',
+    run_on: 'windows-vsCurrent-large',
+  },
+  {
+    name: 'package-rhel',
+    display_name: 'Package RHEL',
+    run_on: 'rhel80-large',
+  },
+  {
+    name: 'package-macos-x64',
+    display_name: 'Package MacOS Intel',
+    run_on: 'macos-1100'
+  },
+  {
+    name: 'package-macos-arm',
+    display_name: 'Package MacOS Arm64',
+    run_on: 'macos-1100-arm64'
+  }
+];
+
+const TEST_PACKAGED_APP_BUILD_VARIANTS = [
+  {
+    name: 'test-packaged-app-ubuntu',
+    display_name: 'Ubuntu 20.04',
+    run_on: 'ubuntu2004-large',
+    depends_on: 'package-ubuntu',
+  },
+  {
+    name: 'test-packaged-app-windows',
+    display_name: 'Windows 10',
+    run_on: 'windows-vsCurrent-large',
+    depends_on: 'package-windows',
+  },
+  {
+    name: 'test-packaged-app-rhel',
+    display_name: 'RHEL 8.0',
+    run_on: 'rhel80-large',
+    depends_on: 'package-rhel',
+  },
+  {
+    name: 'test-packaged-app-macos-11-arm',
+    display_name: 'MacOS arm64 11',
+    run_on: 'macos-1100-arm64-gui',
+    depends_on: 'package-macos-arm'
+  },
+  {
+    name: 'test-packaged-app-macos-11-x64',
+    display_name: 'MacOS x64 11',
+    run_on: 'macos-1100-gui',
+    patchable: false,
+    depends_on: 'package-macos-x64'
+  }
+];
+
+const EOL_SERVER_VERSIONS = [
+  { name: '40x-community', version: '4.0.x' },
+  { name: '40x-enterprise', version: '4.0.x-enterprise' },
+  { name: '42x-community', version: '4.2.x' },
+  { name: '42x-enterprise', version: '4.2.x-enterprise' },
+  { name: '44x-community', version: '4.4.x' },
+  { name: '44x-enterprise', version: '4.4.x-enterprise' },
+  { name: '5x-community', version: '5.x.x' },
+  { name: '5x-enterprise', version: '5.x.x-enterprise' }
+];
+
+// NOTE: the order matters here. The latest element of this array is used
+// to test the packaged app.
```

Comment:
Is it possible to add something to mongodb-runner that would just give us latest stable instead of getting it from this list?

Reply:
I like this idea, not sure if there is a way to test 8.0 though, like the latest next that is not an alpha